src/plugins/discover/public/application/components/sidebar/discover_sidebar.scss

```
@@ -42,54 +42,14 @@
 }
 
 .dscSidebarItem {
-  border-top: 1px solid transparent;
-  position: relative;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  cursor: pointer;
-  font-size: $euiFontSizeXS;
-  border-top: solid 1px transparent;
-  border-bottom: solid 1px transparent;
-  line-height: normal;
-  margin-bottom: $euiSizeXS * 0.5;
-
   &:hover,
-  &:focus {
+  &:focus-within {
     .dscSidebarItem__action {
       opacity: 1;
     }
   }
```

It might be nice to keep the action button visible when the item is **active** as well. You can do this by using an attribute selector that looks for the **presence** of a certain string. Like so:
```suggestion
  &:focus-within,
  &[class*='-isActive'] {
    .dscSidebarItem__action {
      opacity: 1;
    }
  }
```